pkg/plugin/v3/scaffolds/api.go

```diff
@@ -19,14 +19,18 @@ package scaffolds
 import (
 	"fmt"
 
+	"sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/config/api"
+	"sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/config/rbac"
+	"sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/config/samples"
```

Comment:
nit: do we want to group these imports with the other `"sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/*` imports in lines 32-33?